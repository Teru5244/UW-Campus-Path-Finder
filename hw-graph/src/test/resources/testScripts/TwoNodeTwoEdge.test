# Create a graph
CreateGraph g

# Add two nodes
AddNode g n1
AddNode g n2

# Add two edges
AddEdge g n1 n2 e1
AddEdge g n2 n1 e2

# Print the nodes in the graph and the outgoing edges from n1 and n2
ListNodes g
ListChildren g n1
ListChildren g n2