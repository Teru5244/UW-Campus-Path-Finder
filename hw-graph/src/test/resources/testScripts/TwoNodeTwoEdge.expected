# Create a graph
created graph g

# Add two nodes
added node n1 to g
added node n2 to g

# Add two edges
added edge e1 from n1 to n2 in g
added edge e2 from n2 to n1 in g

# Print the nodes in the graph and the outgoing edges from n1 and n2
g contains: n1 n2
the children of n1 in g are: n2(e1)
the children of n2 in g are: n1(e2)